docs/reference/files/charmcraft-yaml-file.rst

```diff
@@ -895,7 +895,7 @@ endpoint.
 .. admonition:: Best practice
     :class: hint
 
-    Always include the ``optional`` key, rather than relying on the default
+    Include the ``optional`` key in all endpoint definitions, rather than relying on the default
     value to indicate that the relation is required. Although this field is
     not enforced by Juju, including it makes it clear to users (and other tools)
     whether the relation is required.
```

Comment:
```suggestion
    Include the ``optional`` key in all endpoint definitions, rather than relying on
    the default value to indicate that the relation is required. Although this field
    is not enforced by Juju, including it makes it clear to users (and other tools)
    whether the relation is required.
```
I think this should fix the lint.